spec/rails_helper.rb

```diff
@@ -80,6 +80,7 @@
   config.include Devise::Test::ControllerHelpers, type: :controller
   config.include Devise::Test::IntegrationHelpers, type: :request
   config.include Capybara::RSpecMatchers, type: :request
+  config.include GovukComponent, type: :helper
```

Comment:
I'm confused by this bit (because I've not seen it before). What's the reason we need it?

Reply:
You are right @cpjmcquillan I thought I needed it for the reference to GovukComponent::HeaderComponent in the spec to work but it isn't required - have removed 👍